tgui/packages/tgui/interfaces/CompostBin.js

```diff
@@ -0,0 +1,97 @@
+import { useBackend, useSharedState } from '../backend';
+import {
+  Button,
+  Section,
+  Box,
+  Flex,
+  Icon,
+  Collapsible,
+  NumberInput,
+  ProgressBar,
+  Dimmer,
+  LabeledList,
+} from '../components';
+import { Window } from '../layouts';
+
+
+export const CompostBin = (props, context) => {
+	const { act, data } = useBackend(context);
+	const { biomass, compost, biomass_capacity, compost_capacity} = data;
+	
+	let [vendAmount, setVendAmount] = useSharedState(context, 'vendAmount', 1);
+	
+  return (
+    <Window>
+      <Window.Content>
+	  <Section label = "Resources">
+      <Flex>
+        <LabeledList>
+        <LabeledList.Item label="Biomass">
+          <ProgressBar
+            value={biomass}
+            minValue={0}
+            maxValue={biomass_capacity}
+            ranges={{
+              good: [biomass_capacity * 0.5, Infinity],
+              average: [biomass_capacity * 0.25, biomass_capacity * 0.5],
+              bad: [-Infinity, biomass_capacity * 0.25],
+            }}
+          >
+            {biomass} / {biomass_capacity} Units
+          </ProgressBar>
+        </LabeledList.Item>	
+		</LabeledList>
```

Comment:
```suggestion
          <LabeledList.Item label="Biomass">
            <ProgressBar
              value={biomass}
              minValue={0}
              maxValue={biomass_capacity}
              ranges={{
                good: [biomass_capacity * 0.5, Infinity],
                average: [biomass_capacity * 0.25, biomass_capacity * 0.5],
                bad: [-Infinity, biomass_capacity * 0.25],
              }}
            >
              {biomass} / {biomass_capacity} Units
            </ProgressBar>
          </LabeledList.Item>	
		  </LabeledList>
```
make sure this looks right in testing, just fix the indentation